src/AlertsManagement/AlertsManagement/AlertCommands/MeasureAzureAlertStatistic.cs

```diff
@@ -0,0 +1,149 @@
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Filter on Monitor Condition")]
+        [PSArgumentCompleter("Fired", "Resolved")]
+        public string MonitorCondition { get; set; }
+
+        /// <summary>
+        /// Severity
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Filter on Severity of alert")]
+        [PSArgumentCompleter("Sev0", "Sev1", "Sev2", "Sev3", "Sev4")]
+        public string Severity { get; set; }
+
+        /// <summary>
+        /// Alert State
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Filter on State of alert")]
+        [PSArgumentCompleter("New", "Acknowledged", "Closed")]
+        public string State { get; set; }
+
+        /// <summary>
+        /// Alert Rule Id
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Filter on Alert Rule Id")]
+        public string AlertRuleId { get; set; }
+
+        /// <summary>
+        /// Time range
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Supported time range values – 1h, 1d, 7d, 30d (Default is 1d)")]
+        [PSArgumentCompleter("1h", "1d", "7d", "30d")]
+        public string TimeRange { get; set; }
+
+        /// <summary>
+        /// Custom time range
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Supported format - <start-time>/<end-time> where time is in ISO-8601 format")]
+        public string CustomTimeRange { get; set; }
+
+        /// <summary>
+        /// Include SmartGroups Count
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   HelpMessage = "Include SmartGroups Count")]
+        [PSArgumentCompleter("true", "false")]
+        public bool IncludeSmartGroupsCount { get; set; }
+
+        #endregion
+
+        protected override void ProcessRecordInternal()
+        {
+            PSAlertsSummary summary = new PSAlertsSummary(this.AlertsManagementClient.Alerts.GetSummaryWithHttpMessagesAsync(
+                groupby: GroupBy,
+                targetResource: TargetResourceId,
+                targetResourceType: TargetResourceType,
+                targetResourceGroup: TargetResourceGroup,
+                monitorService: MonitorService,
+                monitorCondition: MonitorCondition,
+                severity: Severity,
+                alertState: State,
+                alertRule: AlertRuleId,
+                timeRange: TimeRange,
+                customTimeRange: CustomTimeRange,
+                includeSmartGroupsCount: IncludeSmartGroupsCount
+                ).Result.Body);
+
+            // TODO: How to directly call overriden ToString() functions for custom output classes
+            // Defined PSAggregatedCounts in PSSummary
+            WriteObject(summary);
+            WriteObject(summary.AggregatedCounts.ToString());
```

Comment:
You should only write one object to the Output stream here.  You should not write strings.

TRhis eiother means your output object should contain this additional info (it should) and/or that you customize the output format.